### ISSUE TITLE
No longer works with BigSur.  

### ISSUE THREAD

**Issue by FerrisWheelie** [bitgapp/eqMac#902]
One day stopped working and will not reinstall properly.  If I clear everything using the terminal commands you provided I can get it to install and function.  As soon as the computer goes to sleep or reboots it no longer works even though it is shown to be running in Activity Manager.  Once it stops working, nothing can be done to make it function again.

**Comment by smile4yourself** [comment]
I agree. Mine too. I hope he fixes it soon. I have the Pro Lifetime version, but it does not look like it is being kept up to date. I tried contacting him other ways, but never had any success. Soon I am going to stop trying to load it every time my computer is turned on.

**Comment by smile4yourself** [comment]
I should have said I am running Sonoma 14.3.1 on an M1 Ultra

**Comment by FerrisWheelie** [comment]
I have an older Mac that was maxed out at BigSur.  I patched it and am now running Sonoma.  EQ Mac is working but is still glitchy. Sometimes audio just stops; hitting a volume button corrects it until the next occurence.  I have another one running Monterey (unpatched) and it's working fine on that machine.

**Comment by ragauskl** [comment]
> I agree. Mine too. I hope he fixes it soon. I have the Pro Lifetime version, but it does not look like it is being kept up to date. I tried contacting him other ways, but never had any success. Soon I am going to stop trying to load it every time my computer is turned on.

@smile4yourself eqMac is very much maintained (except for public repository), however as complexity of the app grows even what may seem as easy changes take much more time to think through and implement. It's also developed and maintained in all aspects by single person so he has to juggle a lot of work, so I'm temporarily helping to catch up with all the customer support.

**Comment by ragauskl** [comment]
@smile4yourself if I correctly found customer support ticket through eqMac chat it seems like you haven't replied to the last message and were having an issue with eqMac asking to install the driver? Is that still an issue?

**Comment by ragauskl** [comment]
@FerrisWheelie Could you confirm what issues and on which macOS versions are you still experiencing?

**Comment by FerrisWheelie** [comment]
I commented above that I patched my computer and EQmac is now working well with a few with some minor glitches.

**Comment by ragauskl** [comment]
Yeah I've read that, was wondering about those glitches if they were still an issue to know what still needs looking into. Do glitches you're mentioning only include audio stopping with the volume adjustment temporarily resolving the issue?

**Comment by smile4yourself** [comment]
> @smile4yourself if I correctly found customer support ticket through eqMac chat it seems like you haven't replied to the last message and were having an issue with eqMac asking to install the driver? Is that still an issue?

Thanks for your support. I no longer have problems with it, and am running Sonoma 14.5. Best wishes to everyone involved.
